src/update-feeds/update-feeds.ts

```diff
@@ -162,7 +164,154 @@ export const updateFeeds = async (_batch: ReturnType<typeof getFeedsToUpdate>, _
   // batch, execute
 };
 
-export const processBatch = async (batch: ReadDapiWithIndexResponse[], providerName: Provider, chainId: string) => {
+export const deepCheckFeeds = async (
+  batch: ReturnType<typeof getFeedsToUpdate>,
+  providerName: Provider,
+  chainId: ChainId
+) => {
+  const { config } = getState();
+  const chain = config.chains[chainId]!;
+  const { providers, contracts } = chain;
+
+  const provider = new ethers.providers.StaticJsonRpcProvider(providers[providerName]);
+  const server = getApi3ServerV1(contracts.Api3ServerV1, provider);
+  const voidSigner = new ethers.VoidSigner(ethers.constants.AddressZero, provider);
+
+  // We surely have to chunk these? they'll get big.
```

Comment:
The only reason the update can fail is the timestamp check https://github.com/api3dao/airnode-protocol-v1/blob/fa95f043ce4b50e843e407b96f7ae3edcf899c32/contracts/api3-server-v1/DataFeedServer.sol#L121

I think we should just get all the beacon values of this beacon set (using `readDataFeedWithId`). These are very simple so they can be multicalled together for the whole batch (I verified this by multicalling 10K `readDataFeedWithId` on Infura and the limits we are talking here are way smaller).

Reply:
Actually the `dataFeeds` function doesn't make the `require` call so it's even "cheaper" so we might call that.